### PR TITLE
Add Weave Content

### PR DESCRIPTION
**Pull request by galal-hussein**
https://github.com/rancher/docs/issues/1008

### CODE REVIEW

File: content/rancher/v2.x/en/faq/networking/cni-providers/_index.md [hunk 1]
```diff
@@ -91,6 +91,20 @@ For more information, see the following pages:
 - [Project Calico GitHub Page](https://github.com/projectcalico/calico)
 
 
+#### Weave
+
+![Weave Logo]({{< baseurl >}}/img/rancher/weave-logo.png)
```
Comment (alena1108) [inline content/rancher/v2.x/en/faq/networking/cni-providers/_index.md]: we should mention somewhere that Weave support is `Available as of v0.2.0`

File: content/rancher/v2.x/en/faq/networking/cni-providers/_index.md [hunk 2]
```diff
@@ -91,6 +91,20 @@ For more information, see the following pages:
...
+
+Weave Net creates a virtual network that connects Docker containers across multiple hosts and enables their automatic discovery. With Weave Net, portable microservices-based applications consisting of multiple containers can run anywhere, Weave works as a cni plugin to Kubernetes to connect pods across Kubernetes nodes.
+
+Weave support encrypting traffic between peers by assigning a password as a config option or `WEAVE_PASSWORD` environment variable to the weave Kubernetes YAML.
```
Comment (alena1108) [inline content/rancher/v2.x/en/faq/networking/cni-providers/_index.md]: We need to mention that Rancher UI generates WEAVE_PASSWORD so encryption is turned on by default. And perhaps explain a way to reset a password to a custom value - via cluster.yml

**Comment by deniseschannon** [comment]
Can you rebase off of the `staging` branch @galal-hussein

File: content/rancher/v2.x/en/faq/networking/cni-providers/_index.md (hunk 1 above)
Comment (galal-hussein) [reply content/rancher/v2.x/en/faq/networking/cni-providers/_index.md]: fixed

**Comment by alena1108** [comment]
LGTM

**Pull request merged**